runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/diaries/KourendDiaryRequirement.java

```diff
@@ -0,0 +1,135 @@
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.achievementdiary.diaries;
+
+import net.runelite.api.Favour;
+import net.runelite.api.Quest;
+import net.runelite.api.Skill;
+import net.runelite.client.plugins.achievementdiary.GenericDiaryRequirement;
+import net.runelite.client.plugins.achievementdiary.SkillRequirement;
+import net.runelite.client.plugins.achievementdiary.QuestRequirement;
+import net.runelite.client.plugins.achievementdiary.FavourRequirement;
+
+public class KourendDiaryRequirement extends GenericDiaryRequirement
+{
+	public KourendDiaryRequirement()
+	{
+		//EASY
+		add("Mine some Iron at the Mount Karuulm mine.",
+			new SkillRequirement(Skill.MINING, 15));
+		add("Steal from a Hosidius Food Stall.",
+			new SkillRequirement(Skill.THIEVING, 25),
+			new FavourRequirement(Favour.HOSIDIUS, 15));
+		add("Browse the Warrens General Store.",
+			new QuestRequirement(Quest.THE_QUEEN_OF_THIEVES, true));
+		add("Enter your Player Owned House from Hosidius.",
+			new SkillRequirement(Skill.CONSTRUCTION, 25));
+		add("Create a Strength potion in the Lovakengj Pub.",
+			new SkillRequirement(Skill.HERBLORE, 12));
+		add("Fish a Trout from the River Molch.",
+			new SkillRequirement(Skill.FISHING, 20));
+
+		//MEDIUM
+		add("Travel to the Fairy Ring south of Mount Karuulm.",
+			new QuestRequirement(Quest.FAIRYTALE_II__CURE_A_QUEEN, true));
+		add("Use Kharedst's memoirs to teleport to all five cities in Great Kourend.",
+			new QuestRequirement(Quest.THE_DEPTHS_OF_DESPAIR),
+			new QuestRequirement(Quest.THE_QUEEN_OF_THIEVES),
+			new QuestRequirement(Quest.TALE_OF_THE_RIGHTEOUS));
+		//  new QuestRequirement(Quest.THE_FORSAKEN TOWER),
+		//  new QuestRequirement(Quest.THE_ASCENT_OF_ARCEUUS));
+		add("Mine some Volcanic sulphur.",
+			new SkillRequirement(Skill.MINING, 42));
+		add("Enter the Farming Guild.",
+			new SkillRequirement(Skill.FARMING, 45));
+		add("Switch to the Necromancy Spellbook at Tyss.",
+			new FavourRequirement(Favour.ARCEUUS, 60));
+		add("Repair a Piscarilius crane.",
+			new SkillRequirement(Skill.CRAFTING, 30));
+		add("Deliver some intelligence to Captain Ginea.",
+			new FavourRequirement(Favour.SHAYZIEN, 40));
+		add("Catch a Bluegill on Molch Island.",
+			new SkillRequirement(Skill.FISHING, 43),
+			new SkillRequirement(Skill.HUNTER, 35));
+		add("Use the boulder leap in the Arceuus essence mine.",
+			new SkillRequirement(Skill.AGILITY, 49));
+		add("Mine some Volcanic sulphur.",
```

Comment:
This requires 42 mining. I can't find any mention of it requiring 50 firemaking?

Reply:
What?

Reply:
https://oldschool.runescape.wiki/w/Volcanic_sulphur

Reply:
This seems to be misnamed, as the Volcanic Sulphur task is on line 63

Reply:
I'll jump on computer and double check all my spelling/mistakes again was real tired when i wrote these. Thanks for pointing these out if theres anymore let me know.